doc/md_src_files/082_relatedSystems.md

```diff
@@ -105,3 +105,74 @@ In that sense, it is compatible with any of methods mentioned above or SSVC.
+Namely, each list has a different view and vantage on the problem, which makes them appear to be different, but each list does a good job at accurately representing its particular vantage at a point in time.
+
+
+[*System Exposure*](#system-exposure) should be informed by the various scanning platforms such as and Shodan and Shadowserver.
+A service on a device should be scored as [*open*](#system-exposure) if such a general purpose Internet scan finds that the service responds.
+Such scans do not find all [*open*](#system-exposure) systems, but any system they find should be considered [*open*](#system-exposure).
+Scanning software, such as the open-source Nessus, could be used to scan for connectivity inside an organization to catalogue what devices should be scored [*controlled*](#system-exposure) if, say, the scan finds them on an internal network where devices regularly connect to the Internet.  
+
+
+There are some information sources that are not exactly suited to make SSVC decisions, but with a bit of work could be used directly.
+Three prominent examples of this are CVSS impact base metrics, CWE, and CPE.
+
+[*Technical Impact*](#technical-impact) is directly related to the CVSS impact metric group.
+However, this metric group cannot be directly mapped to [*Technical Impact*](#technical-impact) in CVSSv3  because of the Scope metric.
+[*Technical Impact*](#technical-impact) is only about adversary control of the vulnerable component.
+If the value of Scope is Changed in CVSSv3, then the impact metrics are the maximum of the impact on the vulnerable component and other components in the environment.
+As long as the impact metrics in CVSS are clearly about just the vulnerable component, if confidentiality, integrity, and availability metrics are all "high" then [*Technical Impact*](#technical-impact) is [*total*](#technical-impact).
+However, the other values of the CVSSv3 impact metrics cannot be mapped directly to [*partial*](#technical-impact) because of CVSSv3.1 scoring guidance.
+Namely, "only the increase in access, privileges gained, or other negative outcome as a result of successful exploitation should be considered" [@cvss_v3-1].
+The example given is that if an attacker already has read access, but gains all other access through the exploit, then read access didn't change and the confidentiality metric score should be "None".
+However, in this case, SSVC would expect the decision point to be evaluated as [*total*](#technical-impact) because as a result of the exploit the attacker gains total control of the device, even though they started with partial control.
+
+As mentioned in the discussion of [*Exploitation*](#exploitation), [CWE](https://cwe.mitre.org/) could be used to inform one of the conditions that satisfy [*proof of concept*](#exploitation).
+For some classes of vulnerabilities, the proof of concept on how to exploit them is well known and already part of open-source tools.
```

Comment:
Maybe "the method of exploitation" instead of "the proof of concept on how to exploit them"? It seems awkward as written.

Reply:
how's this?

Reply:
Not sure what "this" is referring to...?

Reply:
assuming you meant 51559c0ecfa7cd5f9d701e6a98a0c20a7bb5fc05, yes, that covers it